gigascience/gigadb-website
# A11y - Contact page - heading levels

# Pull request for issue: #1376 

This is a pull request for the following functionalities:

* Wrap contact details in `<address>`
* Fix heading hierarchy

Note: color contrast issues have already been fixed in another PR

## How to test?

* Use AXE chrome extension to review heading levels
* Inspect address and see it's wrapped in `<address>`

## How have functionalities been implemented?

* Used hierarchical heading levels starting with h1, and relied on css classes to define heading style, decoupling semantics from style

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
